tools/webpack-make

```diff
@@ -11,11 +11,14 @@ const argparse = require("argparse");
 const fs = require("fs");
 const child_process = require("child_process");
 
+// argv0 is node
+const webpack_watch = process.argv[1].includes('webpack-watch');
+
 const parser = argparse.ArgumentParser()
 parser.add_argument('-c', '--config', { help: "Path to webpack.config.js", default: "webpack.config.js" })
 parser.add_argument('-r', '--rsync', { help: "rsync webpack to ssh target after build", metavar: "HOST" })
-parser.add_argument('-w', '--watch', { action: 'store_true', help: "Enable webpack watch mode" })
-parser.add_argument('--no-eslint', { action: 'store_true', help: "Disable eslint linting" })
+parser.add_argument('-w', '--watch', { action: 'store_true', help: "Enable webpack watch mode", default: webpack_watch })
+parser.add_argument('--no-eslint', { action: 'store_true', help: "Disable eslint linting", default: webpack_watch })
```

Comment:
I wonder if we should enable eslint by default for watching as well. Personally I find it sometimes useful, sometimes it gets in the way -- it's not really about the time (incremental builds with eslint don't take much longer), but more about whether you are hacking on something where "unused function" etc. is annoying. For consistency I'd generally enable it, and provide an option to disable it. @KKoukiou , @mvollmer , @marusak , do you have an opinion about changing the default?

Also, could this get a short option, like `-e`, as it is rather common?